### PR TITLE
For #172: Started reservation view tests

### PR DESCRIPTION
**Pull request by paulodamaso**
For #172: Started reservation view tests
- Created `Reservation` view module
- Created file for `edit_reservation` screen
- Created test for `edit_reservation` screen
- Left puzzle for continuing implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#199](https://github.com/timelesslounge/timelessis/issues/199) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#199](https://github.com/timelesslounge/timelessis/issues/199) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/bba0a9e7-ca09-4755-aa8e-bd5e081d0ed0); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/restaurants/reservations/views.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+"""
+
+
+from flask import (
+    Blueprint, flash, redirect, render_template, request, url_for
+)
+
+
+bp = Blueprint("reservations", __name__, url_prefix="/reservations")
```
Comment (vladarefiev) [inline timeless/restaurants/reservations/views.py]: @paulodamaso I don't see where did you register this blueprint to app.

File: timeless/restaurants/reservations/views.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+@bp.route("/")
+def list_reservations(reservations):
```
Comment (vladarefiev) [inline timeless/restaurants/reservations/views.py]: @paulodamaso Add pls puzzle to refactor this when GenericViews will be implemented
One of 
https://github.com/timelesslounge/timelessis/issues/173
https://github.com/timelesslounge/timelessis/issues/134

File: timeless/templates/restaurants/reservations/create_edit.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{% extends 'base.html' %}
```
Comment (vladarefiev) [inline timeless/templates/restaurants/reservations/create_edit.html]: @paulodamaso Take a look how do the same in this PR - https://github.com/timelesslounge/timelessis/pull/209/files
It's a flask way. If you can't do it now, let's leave a puzzles:
- add forms for view
- after forms implementation refactor template is that way - https://github.com/timelesslounge/timelessis/pull/209/files

File: tests/view/test_reservations.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import datetime
+import pytest
+from timeless.customers.models import Customer
```
Comment (vladarefiev) [inline tests/view/test_reservations.py]: @paulodamaso according to PEP8 should be one blank line between app modules and thirty-part modules

**Comment by vladarefiev** [comment]
@paulodamaso Take a look on comments pls

**Comment by paulodamaso** [comment]
@vladarefiev Thanks, please take a look to the corrections;

**Comment by vladarefiev** [comment]
@paulodamaso great

**Comment by vladarefiev** [comment]
@rultor merge

File: timeless/templates/restaurants/reservations/create_edit.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+{% from "_formhelpers.html" import render_field %}
+
```
Comment (emilianodellacasa) [inline timeless/templates/restaurants/reservations/create_edit.html]: @paulodamaso please add tag for cache

**Comment by emilianodellacasa** [comment]
@paulodamaso Just a single comment

**Comment by paulodamaso** [comment]
@emilianodellacasa Done, please take a look

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 23 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#199](https://github.com/timelesslounge/timelessis/issues/199) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +25 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**